### ISSUE TITLE
Extract single frame from video

### ISSUE THREAD

**Issue by jmmaxwell** [sbis04/video_trimmer#61]
Is it possible to extract a single frame from a video using this trimmer? If not, is that an enhancement you would consider adding? Regardless, thanks for your excellent package!

**Comment by sbis04** [comment]
I think what you want to get is a single image of a particular frame from the video. Then, for that purpose, there is already a plugin available: [video_thumbnail](https://pub.dev/packages/video_thumbnail).

**Comment by jmmaxwell** [comment]
Hi Souvik, thanks for your response! I don't actually need to create video thumbnails (though thank you for pointing me toward that package). I'm currently using the trimmer to allows users to extract multiple short GIFs from a single long video. I'm interested in allowing them to extract many images from that same video just like I'm allowing them to extract GIFs. Does that make sense?

**Comment by sbis04** [comment]
Yes, I understood. I believe that can also be done using the `video_thumbnail` plugin. You can define the time (in ms) from where you want to take the image, and set the image quality to 100%, so it will generate a high-quality image. So, you will get images generated from your defined timestamp.

If there is already an implementation of this, I think it's not wise to do it again.

Actually, I am also using that plugin to show the preview of the video (in form of images) in the `VideoEditor` of my plugin.

**Comment by qbait** [comment]
What about the picker? Can we configure `VideoEditor` to be a picker for the single frame?

**Comment by sbis04** [comment]
@qbait @jmmaxwell okay, I understood what you guys want to tell. The `video_thumbnail` plugin provides the functionality but there is no UI component like `VideoEditor` in that.

Currently, this plugin can't be configured to pick a single frame, because that will also require some changes in the UI of the `TrimEditor`.

Though I didn't have any such enhancement in the timeline. I will try to implement that once I get some time (probably by the end of December)

**Comment by jmmaxwell** [comment]
Hi @sbis04, I was just curious if you had a chance to work on this at all? Again, thanks for your excellent work on this!

**Comment by SPlus91** [comment]
Similar problem here... it would be great if was implemented a way to show a specific frame of the video in the videoViewer.

**Comment by sbis04** [comment]
Closing this since there's no plan to work on this enhancement.
